cloud/deploy/deploy.go

```diff
@@ -248,6 +251,11 @@ func Deploy(deployInput InputDeploy, client astro.Client) error { //nolint
 			fmt.Sprintf("\n Deployment View: %s", ansi.Bold(deploymentURL)) +
 			fmt.Sprintf("\n Airflow UI: %s", ansi.Bold(deployInfo.webserverURL)))
 	} else {
+		envFileExists, _ := fileutil.Exists(deployInput.EnvFile, nil)
+		if !envFileExists {
+			return envFileMissing
```

Comment:
nit: consider wrapping the error with the `deployInput.EnvFile` for a better UX. For e.g. below:

`return fmt.Errorf("%w %s", envFileMissing, deployInput.EnvFile)`